meetings/2023-02-01.md

```diff
@@ -50,36 +57,125 @@ URL: https://blog.gitter.im/2023/01/16/gitter-is-going-fully-native-matrix-in-fe
 
 ### Too Slow: Please provide Updates-Via shortcut to secure web socket address
 URL: https://github.com/solid/notifications/issues/110
 
 * SC: Call for response to questions 1-5 in comment: https://github.com/solid/notifications/issues/110#issuecomment-1406336400
 * SC: Let me know if clarification needed.
 * SC: Resolving this issue may help with the TR/2022/notifications-protocol-20221231 release, and moving that forward can help with the TR/2022/protocol-20221231 release:
-
-#### Add TR/2022/notifications-protocol-20221231
-URL: https://github.com/solid/specification/pull/491
-
-
-#### Add TR/2022/protocol-20221231
-URL: https://github.com/solid/specification/pull/492
-
-
+  * https://github.com/solid/specification/pull/491
+  * https://github.com/solid/specification/pull/492
+* TBL: You said to look at two questions. Is one of those "is ??? always bad"? and the other one was about caching issues with metadata? 
+* SC: I can summarize. As I understood how people's assumptions are, I see no security differences with either two approaches. I am asking for people to point to me why is one less secure. It seems one of the approaches has caching issues that need to be taken into account. The HEAD response seems to have more of a caching issue than POST on a subscription service. No security differences, yes a difference in caching. 
+* TBL: I disagree that there is no difference when you have sensitive information in the URL. Anytime the client contacts the server sensitive information will be coming back ??? 
+* SC: Both are authorized requests, if either are not they will get a 403. 
+* TBL: These happen thousands of times in option A and much less in option B. 
+* eP: Is this a roadblock for the PR or something we want to resolve for next version? 
+* SC: It is not but hoping it can help move. The PR is there and there are zero reviews. Only issue I can find to move the PR is either accept PR for next release or refuse this publication. People have different positions. If they want to battle out how it should work they should hash it out and come back to me. I don't see any security differences. The questions are there. Just need to be responded. 
+* SC: If yo;'re unauthorized to read ??? you will get a 403 and in that 403 you should not see the ??? revealing 
+* TBL: Right. The profile is topic resource. Aaron is worried about leakage of info because every authorized fetch gets ???
+* SC: Whether the URL is the same for every response..
+* TBL: Sensitive information goes too many times over the net for Aaron's liking. 
+* SC: That is not in the thread. Going over the wire, whether under TLS and so on, is exposed more.. that information is not in that thread so if that is something we need to factor in when deciding whether to accept or reject approach A, I'm fine with that. I just need to know. 
+* eP: We shouldn't make it a roadblock. My strong preference is we go with PR since this is an optimization. Not breaking anything in the PR. 
```

Comment:
```suggestion
* SC: That is not in the thread. Going over the wire, whether under TLS and so on, is exposed more. That information is not in that thread, but if that is something we need to factor in when deciding whether to accept or reject approach A, I'm fine with that. I just need to know. 
* eP: We shouldn't make it a roadblock. My strong preference is we go with the PR, since this is an optimization. Not breaking anything in the PR. 
```